lib/src/test/kotlin/starknet/utils/DevnetClient.kt

```diff
@@ -1,7 +1,9 @@
 package starknet.utils
 
 import starknet.data.types.Felt
+import java.nio.file.Path
 import java.util.concurrent.TimeUnit
+import kotlin.io.path.absolutePathString
 
 const val HEX_ADDRESS_LENGTH = 66
```

Comment:
It can be removed now

Reply:
This is weird. because it needs to be imported even though it looks like a method on Path object. It seems like its because `Path` is `java.nio` object and absolutePathString is a kotlin inline function.

Reply:
actually I meant `const val HEX_ADDRESS_LENGTH = 66` as it's not used now